src/shared/lib/validation/opportunity/sprint-with-us.ts

```diff
@@ -46,14 +46,39 @@ export function validateSWUOpportunityPhaseMaxBudget(raw: number, maxTotalBudget
 interface ValidatedCreateSWUOpportunityPhaseBody extends Omit<CreateSWUOpportunityPhaseBody, 'startDate' | 'completionDate'> {
   startDate: Date;
   completionDate: Date;
+  requiredCapabilities: CreateSWUOpportunityPhaseRequiredCapabilityBody[];
+}
+
+export function validateFullTime(raw: any): Validation<boolean> {
+  return isBoolean(raw) ? valid(raw) : invalid(['You must provide a boolean value.']);
+}
+
+export function validatePhaseRequiredCapabilities(raw: any): ArrayValidation<CreateSWUOpportunityPhaseRequiredCapabilityBody, CreateSWUOpportunityPhaseRequiredCapabilityErrors> {
+  if (!isArray(raw)) { return invalid([{ parseFailure: ['Please provide an array of required capabilities.'] }]); }
+  return validateArrayCustom(raw, validatePhaseRequiredCapability, {});
+}
```

Comment:
Validate that at least one capability is provided.

Reply:
Done, thanks.